dashboard/src/features/projects/overview/components/OverviewTopBar/OverviewTopBar.tsx

```diff
@@ -85,7 +85,7 @@ export default function OverviewTopBar() {
               <div className="mt-1 inline-grid grid-flow-col items-center justify-start gap-2 md:mt-0">
                 <Chip
                   size="small"
-                  label={isPro ? 'Pro' : 'Starter'}
+                  label={currentProject.plan.name}
                   color={isPro ? 'primary' : 'default'}
```

Comment:
plan is `null` when running the local dashboard. I suggest doing optional chaining `currentProject.plan?.name` and maybe add a default

Reply:
this isn't rendered on the local page though